bin/addons-linter

```diff
@@ -4,7 +4,13 @@ var path = require('path');
 
 var absoluteAppRoot = path.resolve(path.join(__dirname, '..'));
 global.appRoot = path.relative(process.cwd(), absoluteAppRoot);
-
-require('../dist/addons-linter')
-  .createInstance({runAsBinary: true})
-  .run();
+try{
+    require('../dist/addons-linter')
```

Comment:
@tofumatt  yes I used the command `npm test` to check the style errors and I didn't receive any error message regarding this.